tools/generate_schema_wrapper.py

```diff
@@ -894,6 +895,44 @@ def generate_config_typed_dicts(fp: Path, /) -> Iterator[str]:
     yield CONFIG_TYPED_DICT.format(typed_dict_args="\n    ".join(top_dict_annotations))
 
 
+def is_relevant_schema(info: SchemaInfo, /) -> bool:
+    EXCLUDE = {"ExprRef", "RelativeBandSize", "ParameterPredicate"}
+    return bool(
+        info.ref
+        and info.refname not in EXCLUDE
+        and info.properties
+        and info.type == "object"
+        and not info.is_value()
+        and "field" not in info.required
+        and not (iskeyword(next(iter(info.required), "")))
+    )
+
+
+def iter_children(info: SchemaInfo, /) -> Iterator[SchemaInfo]:
+    yield from info.properties.values()
+    yield from info.anyOf
+    if info.items:
+        yield info.child(info.items)
```

Comment:
# TODO

Find a more accurate name for what this is doing and add as `SchemaInfo.iter_...` (instance method).

I don't like the overlap with `SchemaInfo.child`, since they aren't really related